src/tests/hitch_test.sh

```diff
@@ -221,6 +231,42 @@ hitch_hosts() {
 	fail "none of supported lsof, sockstat or fstat available"
 }
 
+#-
+# Usage: hitch_worker_hosts
+#
+# Print a list of hosts of the children processes.
+
+hitch_worker_hosts() {
+	HITCH_WORKER_PIDS="$(hitch_worker_pids)"
+	for WORKER_PID in $HITCH_WORKER_PIDS
+	do
+		hosts_of_pid $WORKER_PID
+	done
+}
+
+#-
+# Usage: hitch_hosts
+#
+# Print a list of hosts of the parent or children processes, depending on the
+# availabilty of SO_REUSEPORT_WORKS. In which case the port initialization
+# is done in the worker processes.
+
+hitch_hosts() {
+	SO_REUSEPORT_WORKS=false
+	if test "$(python -c 'import socket; print(1 if hasattr(socket, "SO_REUSEPORT") else 0)')" = "1"
```

Comment:
So this pulls in Python as test dependency, are you sure?
A shorter and simpler idiom would be

    if python -c 'import socket; exit(not hasattr(socket, "SO_REUSEPORT"))'

or

    if ! python -c 'import socket; exit(hasattr(socket, "SO_REUSEPORT"))'

depending on your taste.